qemu_v8.xml

```diff
@@ -6,7 +6,7 @@
         <remote name="arm-gitlab"       fetch="https://git.gitlab.arm.com/" />
 
         <include name="common.xml" />
-        <project path="build"                name="OP-TEE/build.git">
+        <project path="build"                name="xen-troops/optee_build.git" revision="zephyr-optee">
```

Comment:
There is no branch `zephyr-optee` in the mentioned repo. Only `zephyr-test`.

Reply:
Aaah, thanks. Fixed that.